airflow/models/taskmixin.py

```diff
@@ -236,14 +237,14 @@ def set_upstream(
         self._set_relatives(task_or_task_list, upstream=True, edge_modifier=edge_modifier)
 
     @property
-    def downstream_list(self) -> Iterable["DAGNode"]:
+    def downstream_list(self) -> Iterable["Operator"]:
```

Comment:
Oh I don’t think you can do this… Many had tried this before and failed 🙂

Reply:
Interesting, [get_task](https://github.com/apache/airflow/blob/a901ad93faf80f0f4399154ac52920f9da1316e7/airflow/models/dag.py#L2232) is apparently returning an Operator?

Reply:
Yes but… well this thing is a mess.

Reply:
@uranusjr Does this need changing or is it fine as it is?